floydker/src/floydker/render.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python
+# -*- coding:utf-8 -*-
+
+import click
+import yaml
+import jinja2
+import os
+import click_log
+import logging
+import copy
+
+logger = logging.getLogger(__name__)
+
+
+class FilesLoader(jinja2.BaseLoader):
+    """
+    FilesLoader takes a list of template files instead of search path
```

Comment:
Ah, now I see that this extends the jinja BaseLoader class. I'm assuming the base loader uses the search path? Maybe explain that here.